packages/protocol/contracts/rewards/StakingRewards.sol

```diff
@@ -407,6 +460,19 @@ contract StakingRewards is ERC721PresetMinterPauserAutoIdUpgradeSafe, Reentrancy
     return leverageMultiplier;
   }
 
+  /// @notice Calculate the exchange rate that will be used to convert the original staked token amount to the
+  ///   `baseStakingToken()` amount
+  /// @param positionType Type of the staked postion
+  function getBaseTokenExchangeRate(StakedPositionType positionType) public view virtual returns (uint256) {
+    if (positionType == StakedPositionType.CurveLP) {
+      return config.getFiduUSDCCurveLP().getVirtualPrice() / config.getSeniorPool().sharePrice();
+    } else if (positionType == StakedPositionType.Fidu) {
+      return 1;
```

Comment:
We should use an integer representation that includes decimal places, rather than taking the integer to be a whole number. Have a look at how we use `MULTIPLIER_DECIMALS` for examples

Reply:
Updated